src/NServiceBus.Core/Hosting/Helpers/AssemblyScannerConfiguration.cs

```diff
@@ -24,7 +24,7 @@ public class AssemblyScannerConfiguration
         /// <summary>
         /// Defines whether nested directories should be included in the assembly scanning process. Default value is <code>false</code>.
         /// </summary>
-        public bool ScanAssembliesInNestedDirectories { get; set; } = false;
+        public bool ScanAssembliesInNestedDirectories { get; set; }
```

Comment:
> but in that case it should be consistent across the file

I have made one change that gets the code to a more consistent state. if u want to do a full review of that file, and all others, then go ahead. but making a change that matches the standard in the code base should not block a PR because some other part of the code is also inconsistent.